homeassistant/components/plugwise/climate.py

```diff
@@ -89,9 +88,7 @@ def target_temperature(self) -> float | None:
     @property
     def hvac_mode(self) -> str:
         """Return HVAC operation ie. heat, cool mode."""
-        if (mode := self.device.get("mode")) is None or mode not in self.hvac_modes:
-            return HVAC_MODE_OFF
-        return mode
+        return self.device.get("mode")
```

Comment:
It must always return a valid HVAC mode.

Reply:
Yes, that's handled by the backend.

Reply:
Right but is now can return None, because "get()" is used (hence Mypy complains)

Reply:
Wait, what are you saying? Don't use get()? Using `self.device["mode"]` will work?

Reply:
yes, that works. Is that acceptable?